core/src/main/java/io/questdb/cairo/wal/seq/SequencerMetadata.java

```diff
@@ -184,6 +184,10 @@ public void switchTo(Path path, int pathLen) {
         syncToMetaFile();
     }
 
+    public void syncToDisk() {
+        metaMem.sync(true);
```

Comment:
msync + MS_ASYNC is a no-op on Linux, so if we remove this call nothing changes, at least on Linux.

Reply:
changed to 
```
metaMem.sync(false);
```